index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Medini Website Design</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <section class="container">
+        <a><h3 class="main_heading">My Logo</h3></a>
+        <nav>
+            <img class="menubutton" src='./images/Union.svg' alt="menubutton">
+        </nav>
+    </section>
+    <section class="container mobilepara">
+        <h1>I'm Medini<br>Glad to see<br>you!</h1>
+        <p>I'm a software developer! I can help you
+            build a product, feature or website Look
+            through some of my work and 
+            experience! If you like what you see and
+            have a project you need coded, don't
+            hesitate to contact me.</p>
+        <h4>LET'S CONNECT</h4>
```

Comment:
Change the font family:
- This is your requested style:


![Screen Shot 2021-09-25 at 17 38 22](https://user-images.githubusercontent.com/22258274/134787579-ed50da84-3768-415f-8730-5612b866917d.png)

#
Yours:
![Screen Shot 2021-09-25 at 17 39 25](https://user-images.githubusercontent.com/22258274/134787606-63a59381-9ee3-4ea5-8f5c-a1323a7e84ee.png)

#

Reply:
- Should be a `` button`` or ``a`` tag.

Reply:
Changed font family. Also changed this to "a" tag.